### DISCUSSION TITLE
2nd device

### DISCUSSION THREAD

**Discussion by Koobold** [Brikwerk/nxbt#37] [Q&A]
Hello, I would like NXBT to connect to a second device via bluetooth donge, where could I make this setting so that the correct MAC addresses are used, or is it not possible at all?

**Comment by Brikwerk** [comment]
Are you asking if you can create/connect a second controller to a Switch or are you asking if you can connect multiple Switches to a single controller?

Creating multiple controllers and connecting to different Switches is possible. This, however, requires use of a one Bluetooth adapter or dongle per controller.

If you're looking to connect multiple Switches to a single emulated controller, unfortunately, NXBT doesn't support this use-case. In theory, though, I believe this should technically be possible. Implementing this functionality in NXBT, however, would require fairly extensive revision of the controller server.

Overall, if you are aiming to connect to multiple Switches and use NXBT, you'll need to use one Bluetooth dongle per Switch. If there's enough interest in the "multiple Switches for one controller" scenario, I can investigate implementation.
